src/dotnet/ToolManifest/LocalizableStrings.resx

```diff
@@ -126,8 +126,11 @@
   <data name="InvalidManifestFilePrefix" xml:space="preserve">
     <value>Invalid manifest file.</value>
   </data>
-  <data name="MissingVersion" xml:space="preserve">
-    <value>field version is missing.</value>
+  <data name="ManifestVersion0" xml:space="preserve">
+    <value>Manifest version 0 is not supported</value>
+  </data>
+  <data name="ManifestVersionHigherThanSupported" xml:space="preserve">
+    <value>Manifest version is {0}. This manifest may not be supported in this SDK version that can support up to manifest version {1}. Manifest file path: </value>
```

Comment:
We should check this text with @KathleenDollard. Sounds strange as it is.

Reply:
I created https://github.com/dotnet/cli/issues/10081 and I'll see if I can talk to Kathleen next week or see if she can find someone to help